interface/admin.py

```diff
@@ -13,16 +13,21 @@
 
 @admin.register(Assignment)
 class AssignmentAdmin(admin.ModelAdmin):
-    actions = ['download_submissions']
+    actions = ['download_submissions', 'download_all_submissions']
```

Comment:
This part is the same as #46. I suggest using `git rebase --onto master origin/downloadAllSubmissions  StayLoggedIn` to keep only the logged in part. You will get conflicts when we will try to merge this branch after #46, or the other way around.

Reply:
I am not sure if I solved the problem, can you check?

Reply:
It didn't solve it. You can check here to see the differences: https://github.com/vmck/acs-interface/pull/47/files